kratos/containers/variables_list_data_value_container.h

```diff
@@ -109,6 +112,9 @@ class KRATOS_API(KRATOS_CORE) VariablesListDataValueContainer
         : mQueueSize(rOther.mQueueSize), mpCurrentPosition(0),
           mpData(0), mpVariablesList(rOther.mpVariablesList)
     {
+        if(!mpVariablesList) 
```

Comment:
just wonder if we should not throw an error in this case. 
does it make sense to copy an uninitialized varlist?

Reply:
This constructor is called in creating the static nodes for prototype elements among other registered things that do not use the VariableList. So I could not throw an error.